style.css

```diff
@@ -0,0 +1,520 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Lato", sans-serif;
+}
+
+.lato {
+  font-family: "Lato", sans-serif;
+}
+
+.cocogoose {
+  font-family: "cocogoose", sans-serif;
+}
+
+.display-none {
+  display: none;
+}
+
+.color-white {
+  color: #d3d3d3;
+  text-align: center;
+}
+
+.color-black {
+  color: #272a31;
+  text-align: center;
+}
+
+.color-red {
+  color: #ec5242;
+}
+
+#offcanvasNavbar {
+  background: url("./images/homeBack.jpg");
```

Comment:
- [ ] Kindly consider giving this image a transparent background to make your home page an exact duplicate of the provided design it will also improve your page accessibility

Reply:
@divinecharlotte Thank you for helping me find this small and strange bug, apparently, it is something related to the Mac operating system, because in Linux and Windows, it looks good, I already solved it by removing the homonymous image that was being loaded when you opened it from the repository.

**For another code reviewer**, 🙏🏼  I already met with Charlotte and the problem was that there were two images with the same name, one called homeBack.jpg, which is fine and is the one that is referenced in my code, and another called homeback.jpg the that I used when I was testing, for some strange reason apparently Mac OS loads the image that is in lowercase and by default or something like that, so when Charlotte cloned the repository another image came out, not the one I referenced.

The problem was already solved. it should load fine.